wdl2cwl/main.py

```diff
@@ -659,6 +659,19 @@ def convert(workflow: str) -> str:
         output_name = i[1]
         output_glob = get_output(i[2], input_names)
 
+        if "String" in i[0]:
```

Comment:
This conversion is not required everytime a String output is encountered. The conversion is needed for String outputs where the content on the right side of the `=` sign in the WDL task (`i[1]` in this case) contains `read_string`. So the `if` condition here should be modified.